src/bp/nlu-server/typings_v1.d.ts

```diff
@@ -83,7 +83,7 @@ export interface IntentPrediction {
   name: string
   confidence: number
   slots: SlotPrediction[]
-  extractor: 'exact-matcher' | 'classifier'
+  extractor: string
```

Comment:
1. Good point for the component name part. This would really fit with our vision of composite_pipeline.

2. We don't want to fix nlu-server/typings and botpress.d.ts for now